docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/listing/list_item_macros.twig

```diff
@@ -245,18 +244,17 @@
   {% if has_title_heading %}<h3>{% endif %}
   <span class="title">{{- title -}}</span>
   {% if media_link_type == "Infographic" or media_link_type == "Video" %}
-    <span class="media-type">({{ media_link_type|trans }})</span>
+    <span class="usa-collection__mediatype">({{ media_link_type|trans }})</span>
   {% endif %}
   {% if media_link_type == "File" %}
-    <span class="filetype {{file_type}}">
+     <span class="usa-collection__filetype">
       {% if file_type != "unknown" %}
-        {# Note, the spanish should have the word order reversed... #}
-        <span class="show-for-sr">{{ file_type }} {% trans %}file{% endtrans %}</span>
+        [{{file_type}}
       {% else %}
-        {{ file_ext }}<span class="show-for-sr"> {% trans %}file{% endtrans %}</span>
+        [{{ file_ext }}
       {% endif %}
+      , {{ file_size }}]
     </span>
```

Comment:
Looks like you missed https://github.com/NCIOCPL/cgov-digital-platform/pull/4903#discussion_r2325310192